### PR TITLE
Fixing Asciinemas, Logo Position, and adding SCI-F Apps Documentation

### PR DESCRIPTION
**Pull request by vsoch**
This will do the following:

 - add a separate variable "uid" to the asciinema include so that the title can be parsed separately (and include characters like `?` that are illegal for the id attribute.
 - add the start of SCI-F (apps) docs, which was just merged into development!
 - move the logo to be in the sidebar when the page moves.

### CODE REVIEW

**Comment by vsoch** [comment]
hey @GodloveD - when (if) this passes and builds ok, take a look at the sidebar and changes to the asciinema. Let me know your thoughts!

File: assets/css/theme-blue.css [hunk 1]
```diff
@@ -101,3 +101,8 @@ li.sidebarTitle {
     margin-left: 5px;
 
 }
+
+.shiny a:hover {
+    background-color: white !important;
```
Comment (GodloveD) [inline assets/css/theme-blue.css:106]: Ahh.  Beautiful css magic.

**Comment by GodloveD** [comment]
This looks great!  I see you also updated the contributing-docs page so folks will now know how to add asciicasts with the new syntax.  Someday hopefully it will be more than just you and me.  😸

**Comment by vsoch** [comment]
haha yes! But I don't mind smaller development teams so much, you know what they say about cooks and soup :)

**Pull request merged**